duckduckgo/remote-messaging-config
# Remove ppro onboarding part2 promo

Asana: https://app.asana.com/0/1207619243206445/1207546055898912/f

How to test:

- from any internal build
- go to settings, remote messaging framework, and enable staging
- fire button or force a restart
- ensure app opens correct
